README.md

```diff
@@ -58,64 +56,65 @@ $ pelican content --verbose
 Done: Processed 1 articles, 0 drafts, 0 pages, 0 hidden pages and 0 draft pages in 0.17 seconds.
 ```
 
-### SEO report
+### SEO Report
 
-SEO analyzes all your articles and pages, and generate an HTML SEO report in your Pelican root project : `seo-report.html`
+The SEO plugin analyzes all your articles and pages and generate an SEO HTML report in your Pelican project root: `seo-report.html`
 
-Example :
+Example:
 
 ![SEO report example](docs/seo-report-example.png)
 
-You can set up a limit for article and page analysis in the plugin settings.py. By default, it's set up to 10 articles and 10 pages.
+You can set limits for article and page analysis in the plugin settings.py. By default, it's set up to 10 articles and 10 pages.
 
-```
+```python
 ARTICLES_LIMIT = 10
 PAGES_LIMIT = 10
 ```
 
-The analysis works from the most recents articles or pages to the olders according to the `date` metadata.
-Analysis is focused on :
-* Page title `<title></title>`
-* Page description `<meta name="description" content="" />`
-* Title content `<h1></h1>`
-* Internal link `<a href="SITEURL/..."></a>`
+The SEO analysis begins with the most recent articles/pages, according to the `date` metadata, and is focused on the following criteria:
 
-Which are declared by article and page metadata and content :
+* Page title: `<title></title>`
+* Page description: `<meta name="description" content="" />`
+* Heading content: `<h1></h1>`
+* Internal site links: `<a href="SITEURL/..."></a>`
+
+The above information is defined in your source content, such as the following example Markdown file:
 
 ```markdown
-# article.md
-Title: Title page
-Description: Description page
+Title: Page Title
+Description: Page Description
+
+# Heading Content
 
-# Title content
 [Internal link](https://example.com/about.html)
 ```
 
-Theses elements form the basis of page SEO optimization. Have a look to the report, optimize your articles and pages according to analysis, and restart the process to see if all elements are now in green.
```

Comment:
Why remove this sentence `Theses elements form the basis of page SEO optimization.`? I find it interesting to explain why the feature focus on these specific elements.